.ci/sharp_coll_test_wrapper

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -eE
+# Wrapper to add correct parameter to the main scripts without refactoring
+TEST_DEV=$(ibdev2netdev | awk '{ print $1 }' ):1
+./sharp_coll_test -d "${TEST_DEV}" "${@}"
```

Comment:
Is it safe to use such relative paths here?

Reply:
It's ok. The script is copied into dir with tests